data/db.sql

```diff
@@ -132,6 +132,7 @@ CREATE TABLE `be_sessions` (
 
 LOCK TABLES `be_sessions` WRITE;
 /*!40000 ALTER TABLE `be_sessions` DISABLE KEYS */;
+INSERT INTO `be_sessions` VALUES ('cefe16208fb107ae20fd845a777b588e','172.18.0.7',1,1632307483,'a:5:{s:26:\"formProtectionSessionToken\";s:64:\"c0d657839315e0c19f5fd43d1ba9636ceec62be3135f8a06e4a12b4c3d49cf6b\";s:27:\"core.template.flashMessages\";N;s:80:\"extbase.flashmessages.tx_extensionmanager_tools_extensionmanagerextensionmanager\";N;s:49:\"TYPO3\\CMS\\Backend\\Controller\\PageLayoutController\";a:1:{s:12:\"search_field\";N;}s:46:\"extbase.flashmessages.tx_belog_system_beloglog\";N;}',0);
```

Comment:
`db-cleanup.sh` needs to be run before `db-export.sh` (because it cleans up the actual DB). I can add a combined script later.